Jenkinsfile

```diff
@@ -31,7 +31,7 @@ node("docker") {
     repo = 'jfrog-cli'
     sh 'rm -rf temp'
     sh 'mkdir temp'
-    def goRoot = tool 'go-1.20.14'
+    def goRoot = tool 'go-1.22.2'
```

Comment:
checked that exists on the versions yaml?

Reply:
Yes, i've installed it in the new Jenkins and also updated the ecosystem-integration-env

Reply:
great, also 1.22.3 is avaliable now @EyalDelarea